runtime/src/iree-amd-aie/driver/xrt-lite/direct_command_buffer.cc

```diff
@@ -203,6 +207,12 @@ static iree_status_t iree_hal_xrt_lite_direct_command_buffer_dispatch(
     bo->sync(shim_xdna::direction::device2host);
   }
 
+  auto end = std::chrono::high_resolution_clock::now();
```

Comment:
We should be measuring before the bo sync as we also do that on the input side. This way we get the pure AIE kernel time, without any runtime overheads.